enterprise/api/v1/views.py

```diff
@@ -308,11 +311,12 @@ def enroll_learners_in_courses(self, request, pk):
             existing_users = {
                 result.pop('user') for result in results['successes'] if result['course_run_key'] == course_run
             }
+            LOGGER.info("Successfully bulk enrolled learners: {}".format(pending_users | existing_users))
```

Comment:
what does the pipe do here? did you intend to + instead?

Reply:
nvm I think I missed they were dicts

Reply:
you can't add sets together I've learned 🙃 the correct operation is to join them with the `|` character.

Reply:
ie 
```
Python 3.8.5 (default, Jul 21 2020, 10:48:26) 
[Clang 11.0.3 (clang-1103.0.32.62)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> setA = {'a'}
>>> setB = {'b'}
>>> setA | setB
{'b', 'a'}
>>> 
```
and 

```
>>> setA + setB
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for +: 'set' and 'set'
>>> 
```

Reply:
yup good one thanks so sets not dicts yes, thanks for verifying this